config/config.go

```diff
@@ -307,9 +307,10 @@ func ValidateEmail(email string) bool {
 }
 
 func ValidateEndpoints(daemonEndpoint string, passthroughEndpoint string) error {
+
 	passthroughURL, err := url.Parse(passthroughEndpoint)
-	if err != nil {
-		return errors.New("passthrough endpoint must be a valid URL")
+	if err != nil || passthroughURL.Host == "" {
```

Comment:
Lets add the isValidURL check as well?

Reply:
passthroughURL, err := url.Parse(passthroughEndpoint) , this line takes care of it  
isValidURL will just return a boolean , 
and we also need to do additiona checks in the function from the passthroughURL struct 
For example 
the port number after parsing to make sure the daemon port and the service port are not the same